arbiter-core/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 
 pub mod agent;
 pub mod bindings;
+/// Module for managing the environment.
```

Comment:
We don't really need doc comments in the lib, instead we can use module level comments like `//!` (see here: https://doc.rust-lang.org/reference/comments.html)

Reply:
Nice